### ISSUE TITLE
no verb conjugations being returned for hebrew

### ISSUE THREAD

**Issue by robsilva** [s0ftik3/reverso-api#28]
I am running into a strange problem where the getConjugation method returns no conjugation for the Hebrew language.
It works for Russian though

e.g.:
```
 reverso.getConjugation( "לִרְצוֹת", 'hebrew',
        async (err, response) => {
            if (err) throw new Error(err.message)
    
            console.log(response)

           return response;

        }
    )
```
returns:
```
{
data: {
ok: true,
verbForms: [
{
id: 0,
conjugation: "Present",
verbs: [ ]
},
{
id: 1,
conjugation: "Past",
verbs: [ ]
},
{
id: 2,
conjugation: "Future",
verbs: [ ]
},
{
id: 3,
conjugation: "Passive Participle",
verbs: [ ]
}
]
}
}
```

I wonder if it's something to do with the language being RTL right to left

**Comment by s0ftik3** [comment]
it might be due to RTL, yes. It happened before to getContext method. I’ll check out it later and release a patch asap. Thanks!

**Comment by s0ftik3** [comment]
fixed in 2.22.6
